Github action failing to install dependencies

The build on GiHub Actions is failing, due to missing dependencies in the Debian/Ubuntu apt repository.

Stacktrace:
```
  sudo apt-get -yqq install libffi-dev libssl-dev libcurl4-gnutls-dev libxml2-dev libxslt1-dev
  gem install bundler --no-document
  bundle install
  bin/cibuild.sh
  shell: /bin/bash -e {0}
  env:
    RACK_ENV: test
    JEKYLL_ENV: test
    NOKOGIRI_USE_SYSTEM_LIBRARIES: true
E: Failed to fetch http://ppa.launchpad.net/ondrej/php/ubuntu/pool/main/libx/libxml2/libxml2-dev_2.9.9+dfsg-1+ubuntu18.04.1+deb.sury.org+2_amd64.deb  404  Not Found [IP: 91.189.95.83 80]
E: Failed to fetch http://ppa.launchpad.net/ondrej/php/ubuntu/pool/main/libx/libxml2/libxml2_2.9.9+dfsg-1+ubuntu18.04.1+deb.sury.org+2_amd64.deb  404  Not Found [IP: 91.189.95.83 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
##[error]Process completed with exit code 100.
```

--- a/.github/workflows/cibuild.yml
+++ b/.github/workflows/cibuild.yml
@@ -24,7 +24,7 @@ jobs:
         ruby-version: 2.6.x
     - name: Run Test Suite
       run: |
-        sudo apt-get -yqq install libffi-dev libssl-dev libcurl4-gnutls-dev libxml2-dev libxslt1-dev
+        sudo apt-get -yqq install patch zlib1g-dev liblzma-dev libffi-dev libssl-dev libcurl4-gnutls-dev libxslt-dev
         gem install bundler --no-document
         bundle install
         bin/cibuild.sh

